back-end/confirmHealth.js

```diff
@@ -1,3 +1,25 @@
-const confirmHealth = (snack) => {};
+const confirmHealth = (snack) => {
+ if (
+   typeof snack.protein !== 'number' &&
+   typeof snack.fiber !== 'number' &&
+   typeof snack.added_sugar !== 'number'
+ ) {
+   return null;
+ }
+ if (snack.protein >= 5 && snack.added_sugar < 5) {
+   return true;
+ }
+
+ if (snack.fiber >= 5 && snack.added_sugar < 5) {
+   return true;
+ }
+
+ if (snack.protein > 5 && snack.fiber > 5 && snack.added_sugar < 5) {
+   return true;
+ }
```

Comment:
if we start with if(sugar >5) return false we would avoid repetition on added sugar being less than 5